### PR TITLE
(NETTY-10) Shutdown executor handler when shutting down netty service

### PR DESCRIPTION
**Pull request by chtyim**

### CODE REVIEW

File: src/main/java/co/cask/http/NettyHttpService.java [hunk 1]
```diff
@@ -272,14 +273,21 @@ public InetSocketAddress getBindAddress() {
...
         LOG.warn("Timeout when closing all channels.");
       }
     } finally {
       resourceHandler.destroy(handlerContext);
       bootstrap.releaseExternalResources();
+      if (executionHandler != null) {
+        executionHandler.releaseExternalResources();
+        Executor executor = executionHandler.getExecutor();
+        if (executor instanceof ExecutorService) {
+          ((ExecutorService) executor).awaitTermination(CLOSE_CHANNEL_TIMEOUT, TimeUnit.SECONDS);
```
Comment (poornachandra) [inline src/main/java/co/cask/http/NettyHttpService.java]: Don't we need to wait for termination of bossExecutor and workerExecutor pools?

**Comment by poornachandra** [comment]
@chtyim Just one comment, otherwise LGTM.

File: src/main/java/co/cask/http/NettyHttpService.java (hunk 1 above)
Comment (chtyim) [inline src/main/java/co/cask/http/NettyHttpService.java]: Those are done by the `ServerBootstrap.shutdown`.

**Comment by chtyim** [comment]
Squashed commits.

**Pull request merged**